platforms/chibios/converters/elite_c_to_elite_pi/_pin_defs.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// Left side (front)
+#define D3 0U
+#define D2 1U
+//      GND
+//      GND
+#define D1 2U
+#define D0 3U
+#define D4 4U
+#define C6 5U
+#define D7 6U
+#define E6 7U
+#define B4 8U
+#define B5 9U
+
+// Right side (front)
+//      RAW
```

Comment:
This is not RAW on Elite-C, it is B0. Can B0 be mapped to 10U or 11U so that the converter can at least succeed on boards that use B0 on Elite-C? Even if it means the user has to jump the pin...